tests/test_models/test_voxel_encoders/test_voxel_encoders.py

```diff
@@ -15,3 +16,25 @@ def test_hard_simple_VFE():
 
     outputs = hard_simple_VFE(features, num_voxels, None)
     assert outputs.shape == torch.Size([240000, 5])
+
+
+def test_cylindrical_VFE():
+    if not torch.cuda.is_available():
+        pytest.skip('test requires GPU and torch+cuda')
+    cylindrical_VFE_cfg = dict(
```

Comment:
Please make sure that UT covers all the cases as much as possible, such as whether to return  `return_point_feats`.